lib/inspec/metadata.rb

```diff
@@ -35,14 +35,9 @@ def initialize(ref, logger = nil)
     end
 
     def supports(sth, version = nil)
-      params[:supports] ||= []
```

Comment:
Oh sure. Got you. Can we add some inspec.yml tests to increase our confidence :-)